docs.md

```diff
@@ -0,0 +1,37 @@
+ تجربتي مع كراودن (Crowdin)
```

Comment:
```suggestion
 ## تجربتي مع كراودن (Crowdin)
```

Reply:
I liked reading about your experience using Crowdin but the file is plain text without any formatting. You need to use Markdown syntax to format the headings, bulletin points, ...etc.

For example, the header should start with #.  [Here are more details about Markdown syntax](https://www.markdownguide.org/basic-syntax/). Let me know if you have questions!

Reply:
Thank you very much @BatoolMM for your feedback. 
I will make the necessary changes.